Source/WebCore/platform/graphics/egl/GLContextEGL.cpp

```diff
@@ -464,6 +470,51 @@ PlatformGraphicsContext3D GLContextEGL::platformContext()
 }
 #endif
 
+EGLDisplay GLContextEGL::getEGLDisplay() const
+{
+    return m_display.eglDisplay();
+}
+
+EGLConfig GLContextEGL::getEGLConfig() const
+{
+    EGLConfig config = nullptr;
+    if (!getEGLConfig(m_display.eglDisplay(), &config, WindowSurface)) {
+        WTFLogAlways("Cannot obtain EGL window context configuration: %s\n", lastErrorString());
+        config = nullptr;
+    }
+    return config;
+}
+
+EGLSurface GLContextEGL::getEGLSurface() const
+{
+    return m_surface;
+}
+
+EGLContext GLContextEGL::getEGLContext() const
+{
+    return m_context;
+}
+
+unsigned int GLContextEGL::getWindowWidth() const
+{
+    const unsigned int WIDTH = 1920;
+    unsigned int ret_val = WIDTH;
+    char *tmp;
+    if ( (tmp = std::getenv("WPE_INIT_VIEW_WIDTH")) )
```

Comment:
Not optimizing this (into some stored value), since it will be called on average 1 time per lifecycle, so skórka nie warta wyprawki.